spec/controllers/users_controller_spec.rb

```diff
@@ -1,4 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe UsersController, type: :controller do
+  describe 'GET #login' do
+    before { fake_cas }
+
+    it 'redirects to root path by default' do
+      get :login
+      expect(response).to redirect_to root_path
+    end
+
+    it 'redirects to the specified url' do
+      url = '/somewhere'
```

Comment:
Wait, does this work? That's not a legit url, right? Can we just get a comment or something here? If I'm confused and I have some idea of what's going on, any newcomers would definitely be confused.